adr/0004-supported-python-versions.md

```diff
@@ -0,0 +1,36 @@
+# Supported Python Versions
+
+Date: 2017-11-20
+
+## Status
+
+Proposed
+
+## Context 
+
+eLife has numerous projects written completely and partly with the [Python programming language](https://www.python.org/).
+
+In order to provide language version consistency across projects we need to get a consensus on which versions we are going to support. 
```

Comment:
`we have only gone up to Python 3.5 due to the default Python versions pre installed on the Ubuntu distributions we use` goes here

Reply:
Does it? That's worded like a decision. Saying that '16.04 provides 3.5' is context.